hokusai/production.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metaphysics-web
+spec:
+  replicas: 2
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: metaphysics
+        layer: application
+        component: web
+      name: metaphysics-web
+      namespace: default
+    spec:
+      containers:
+        - name: metaphysics-web
+          envFrom:
+          - configMapRef:
+              name: metaphysics-environment
+          image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/metaphysics:production
+          imagePullPolicy: Always
+          ports:
+          - containerPort: 5000
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: metaphysics
+    layer: application
+    component: web
+  name: metaphysics-web
```

Comment:
Should we do`annotations` we did in other apps for forcing `https` stuff here too? 

https://github.com/artsy/lewitt-api/blob/master/hokusai/production.yml#L81

Reply:
Good call.